### PR TITLE
kola/test: add kubernetes 1.24.1

### PR DESCRIPTION
**Pull request by tormath1**
In this PR, we remove the tests for Kubernetes 1.21.10 and we provide a new test for Kubernetes 1.24.1.

## Testing done

In progress: http://jenkins.infra.kinvolk.io:8080/job/os/job/manifest/5816/cldsv/

## Note for reviewers

Still need to decide if we want to exclude kubernetes 1.24.1 from being tested on version != from next major.

- [x] Changelog entries added in the respective `changelog/` directory (user-facing change, bug fix, security fix, update)

### CODE REVIEW

**Review by jepio — APPROVED** [review]
Just out of interest: whats your process to generate the testConfig map entry?

**Comment by tormath1** [comment]
> Just out of interest: whats your process to generate the testConfig map entry?

I use with this gist: https://gist.github.com/tormath1/57f4e3794b2645a7617f50819d6e20e8 to get the SHA signatures :)

**Comment by tormath1** [comment]
On hold - waiting for Alpha release. Otherwise, Kubernetes 1.24.1 tests will fail.

**Comment by tormath1** [comment]
@flatcar-linux/flatcar-maintainers I added two extra commits since the approval:
* https://github.com/flatcar-linux/mantle/pull/337/commits/8e8116bbdbc9b237fa00b39bef55460dcc96ea6a: do not activate SELinux for Flannel CNI
* https://github.com/flatcar-linux/mantle/pull/337/commits/81a29604ab43fc3871f47a4cf426492a2484b574: run Kubernetes 1.24 tests only against the next Alpha release

**Comment by pothos** [comment]
I've backported the changes to enable containerd by default and load the required modules

**Pull request merged**
